### PR TITLE
Updating the p5.js references to 0.7.3

### PR DESCRIPTION
**Pull request by gruselhaus**

### CODE REVIEW

**Comment by gruselhaus** [comment]
@meiamsome Could you double check the rules for the replacement? Four 👀 see more than two ;)

**Comment by gruselhaus** [comment]
@shiffman Should we update the Coding Challenges only or should we update the Tutorials, Courses, ... as well?

**Comment by gruselhaus** [comment]
@shiffman Should we update the Coding Challenges only or should we update the Tutorials, Courses, ... as well?

**Comment by shiffman** [comment]
I'm good with updating everything!

**Comment by gruselhaus** [comment]
Ok. I'm ready to go. Let's wait until @meiamsome reviewed and then merge 🚀

File: package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "0.0.1",
   "description": "Coding Train Website",
   "main": "none",
-  "dependencies": {},
+  "dependencies": {
+    "replace-in-file": "^3.4.4"
```
Comment (meiamsome) [inline package.json]: Perhaps this should be a dev dep instead of a dependency?

Comment (gruselhaus) [reply package.json]: Agreed!

File: p5Update.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+(() => {
+  asyncForEach(options, async option => {
+    try {
+      const changes = await replace(option);
+      console.log(`${option.name}: `, changes.join(", "));
+    } catch (error) {
+      console.error("Error occurred:", error);
+    }
+  });
+})();
```
Comment (meiamsome) [inline p5Update.js]: There should be a new line here ideally!

Any reason for the for each helper? You ought to be able to just use an async IIFE:
```javascript
(async () => {
  for (const option of options) {
    try {
      const changes = await replace(option);
      console.log(`${option.name}: `, changes.join(", "));
    } catch (error) {
      console.error("Error occurred:", error);
    }
  }
})();
```

Comment (gruselhaus) [reply p5Update.js]: Oh, great. Thank you. I'm not that familiar with node.js.

**Comment by prcoy** [comment]
How do I download the files for CC_064??? There is no zip file for them. Do have to download _individual_ files??

**Pull request merged**